subcommands/pkg/plakar-pkg-add.1

```diff
@@ -13,13 +13,19 @@ command adds a local or a remote plugin.
 .Pp
 If
 .Ar plugin
-is an absolute path, or if it starts with
-.Sq ./ ,
-then it is considered a path to a local plugin file, otherwise
-it is downloaded from the Plakar plugin server.
-In the latter case, the user must be logged in via the
+matches an existing local file, it is installed directly.
+Otherwise, it is treated as a recipe name and downloaded from the Plakar plugin
+server which requires a login via the
 .Xr plakar-login 1
 command.
+.Pp
+Installing plugins without logging in is possible via the
+.Xr plakar-pkg-build 1
+command
+.Pq provided you have a working Go toolchain available .
```

Comment:
nitpick: for now all the plugins are written in go, but nothing actually requires this.  The plugins only need to speak grpc, so they can be implemented in a different language.

What do you think if we say instead something along the lines of "provided that you have the build dependencies available"?